mono/utils/mono-threads-windows.c

```diff
@@ -286,10 +287,10 @@ mono_threads_suspend_begin_async_resume (MonoThreadInfo *info)
 		if (!res) {
 			return FALSE;
 		}
-	}
 #else
-	g_error ("Not implemented due to lack of SetThreadContext");	
+		g_error ("Not implemented due to lack of GetThreadContext/SetThreadContext");
```

Comment:
This should only say SetThreadContext. And let's ask for it.

Reply:
I mean, I see GetThreadContext is in UWP current docs and 10240 SDK.
Agreed SetThreadContext is missing.

Reply:
Yes, since SetThreadContext is missing the complete block is ruled out, viewing them as a pair of needed API's.

Reply:
The old version was correct. SetThreadContext is the problem. GetThreadContext is not.

Reply:
Will keep since this we have users supporting older UWP versions and would like to use this file.

Reply:
10240 SDK has GetThreadContext.
SetThreadContext remains a problem though.